pkgs/development/libraries/gdal/default.nix

```
@@ -20,22 +20,22 @@ stdenv.mkDerivation rec {
 
   sourceRoot = "source/gdal";
 
-  nativeBuildInputs = [ autoreconfHook ];
+  nativeBuildInputs = [ autoreconfHook pkg-config ];
 
   buildInputs = [ unzip libjpeg libtiff libpng proj openssl sqlite
-    libspatialite libgeotiff poppler hdf4 qhull giflib expat libxml2 ]
-  ++ (with pythonPackages; [ python numpy wrapPython ])
-  ++ stdenv.lib.optional stdenv.isDarwin libiconv
-  ++ stdenv.lib.optionals netcdfSupport [ netcdf hdf5 curl ];
+    libspatialite libgeotiff poppler hdf4 qhull giflib expat libxml2 postgresql]
+    ++ (with python3Packages; [ python numpy wrapPython ])
+    ++ stdenv.lib.optional stdenv.isDarwin libiconv
+    ++ stdenv.lib.optionals netcdfSupport [ netcdf hdf5 curl ];
 
-  configureFlags = [
+ configureFlags = [
     "--with-expat=${expat.dev}"
     "--with-jpeg=${libjpeg.dev}"
     "--with-libtiff=${libtiff.dev}" # optional (without largetiff support)
     "--with-png=${libpng.dev}"      # optional
     "--with-poppler=${poppler.dev}" # optional
     "--with-libz=${zlib.dev}"       # optional
-    "--with-pg=${postgresql}/bin/pg_config"
+    "--with-pg=yes"		    # since gdal 3.0 doesn't use ${postgresql}/bin/pg_config
```

There was some weird spacing.

```suggestion
    "--with-pg=yes" # since gdal 3.0 doesn't use ${postgresql}/bin/pg_config
```

---

For my local development I am using nixfmt (https://github.com/serokell/nixfmt), but I didn't want to change the whole file. Reading all of your comments seems like it is inevitable. I will do the adjustments tomorrow.

---

Took me a lot of time. I had to actually undo all the commits and make it a new as the plugin I was using in VSCode was adding strange comments. Now it should be OK and all commits make sense.